cmake/onnxruntime_python.cmake

```diff
@@ -355,18 +355,6 @@ if (onnxruntime_USE_TENSORRT)
   )
 endif()
 
-if (onnxruntime_USE_OPENVINO)
-  if(NOT WIN32)
-    add_custom_command(
```

Comment:
does this break python bindings for windows build if you remove this?
seems like the post build logic is only for non WIN32

Reply:
Thanks for pointing out to this. I realized the .dll files were never getting copied to /onnxruntime/capi directory for windows builds even in the first buid and no python sample was working with OpenVINO-EP on windows.

I have made the changes now and this does the thing for us on windows and works on Linux as well.

Earlier, we were using the wrong destination for .so files to get copied to on Linux. so, the post_build step wasn't working right for Linux.